src/main/webapp/scripts/gallery.js

```diff
@@ -7,20 +7,29 @@ console.log("initializing masonry");
 var $grid = $('.grid').masonry({
     itemSelector: '.grid-item',
     percentPosition: true,
-    columnWidth: '.grid-sizer',
```

Comment:
did this not make any difference with the view?

Reply:
nope!